amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -2597,6 +2687,27 @@ RpptDataType getRpptDataType(vx_enum vxDataType) {
     }
 }
 
+RpptLayout getRpptLayout(vxTensorLayout layout) {
```

Comment:
change to const std::map. I suggest moving all common audio functions to a common header file so it is not redefined in every single node

Reply:
Removed the switch statement and used const std::map for layout mapping